tests/integration/targets/setup_mysql/vars/main.yml

```diff
@@ -15,6 +15,7 @@ python_packages: [pymysql == 0.9.3]
 install_prereqs:
   - libaio1
   - libnuma1
+  - libncurses5
```

Comment:
Why is that package needed?

Reply:
the default test container was upgraded to ubuntu2004 as well as some libraries were removed, so collection maintainers should install them manually if they need them. If we remove this line, `dbdeployer` will fail:) I've just checked it once more.